src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -553,6 +554,15 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
             write_to_disk(state_data, output_file_path)
         logger.info(f"Finished writing {observer} files for {state_name}.")
 
+    # Copy over metadata files and subset proportions file
```

Comment:
This comment doesn't seem correct - you're actually reading and writing here, not just copying